src/lib/stores/networksStore.ts

```diff
@@ -0,0 +1,54 @@
+import { writable, type Writable } from 'svelte/store';
+
+/**
+ * Represents the available networks.
+ */
+export enum Network {
+  NONE = 'NONE',
+  TESTNET = 'TESTNET',
+  MAINNET = 'MAINNET',
+  LOCALHOST = 'LOCALHOST',
+  CUSTOM = 'CUSTOM',
+}
+
+/**
+ * Represents information about a network.
+ */
+export interface NetworkInfo {
+  name: string | undefined;
+  endpoint: URL | undefined;
+  genesisHash: string | undefined;
```

Comment:
```suggestion
  name?: string;
  endpoint?: URL;
  genesisHash?: string;
```

Reply:
This is related to the `NONE` and `CUSTOM` keys.

Reply:
Totally get that. I'm just following the preferred way of handling an optional parameter from the typescript docs: https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters

Reply:
I don't see a reference specifically to a Map/Record in that document.

Reply:
It's not about a map, it's just handling the option for a value to be undefined. Typescript typically uses the "?". Yours will still work.

Reply:
I see now.  You were talking the `?` shorthand for `_type_ | undefined`. We can change that.